### PR TITLE
Move flyte types to separate folders

### PR DESCRIPTION
**Pull request by katrogan**

### CODE REVIEW

**Comment by katrogan** [comment]
@wild-endeavor @kumare3

File: flytekit/types/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-from flytekit.types.flyte_file import FlyteFile
-from flytekit.types.pandas_schema import PandasSchemaReader, PandasSchemaWriter
+from flytekit.types.file import FlyteFile
```
Comment (wild-endeavor) [inline flytekit/types/__init__.py:1]: we should clear out this file.

Comment (katrogan) [reply flytekit/types/__init__.py:1]: we need the pandas_schema imports at least to register the pandas types without necessarily importing the package

Comment (wild-endeavor) [reply flytekit/types/__init__.py:1]: But if this file isn't empty, if someone does
```
from flytekit.types.file import FlyteFile
```
All the pandas stuff will be loaded as well.

Comment (kumare3) [reply flytekit/types/__init__.py:1]: ya, for a user to import file, they should need to `from flytekit.types.file import Flytefile`

File: flytekit/types/pandas/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .types import PandasSchemaReader, PandasSchemaWriter
```
Comment (kumare3) [inline flytekit/types/pandas/__init__.py:1]: this is interesting, so after this FlyteSchema will not work? as it needs the PandasSchemaReader/Writer to be present. I actually think PandasSchemaReader/Writer should be in types.schema

**Comment by katrogan** [comment]
PTAL @wild-endeavor @kumare3

File: flytekit/types/pandas/__init__.py (hunk 2 above)
Comment (katrogan) [reply flytekit/types/pandas/__init__.py]: good idea, done

**Review by kumare3 — APPROVED** [review]

**Pull request merged**